articles/logs/streams/azure-event-grid.md

```diff
@@ -0,0 +1,97 @@
+---
+title: Azure Event Grid Log Streams
+description: Learn how to create an event-driven workflow using Azure Event Grid and send your tenant logs anywhere within the Azure ecosystem.
+toc: false
+topics:
+ - logs
+ - streams
+ - event-streams
+contentType: how-to
+---
+
+# Integrate Azure Event Grid with Auth0
+
+Azure’s Event Grid is a serverless event bus that acts as an intermediary allowing you to send data from Auth0 into the Azure ecosystem.
```

Comment:
Maybe something like:

[Azure Event Grid](https://azure.microsoft.com/en-us/services/event-grid/) is a serverless event bus that lets you send event data from any source to any destination.

You can create event-driven workflows using to Event Grid send your Auth0 tenant logs to targets such as Azure Functions, Event Hubs, Sentinel, and Logic Apps.

service for managing routing of all events from any source to any destination.